pegasus/sites.v3/code.org/views/target_languages.haml

```diff
@@ -1,13 +1,11 @@
-- languages = { 'Albanian' => 'sq', 'Arabic' => 'ar', 'Armenian' => 'hy-AM', 'Azerbaijani' => 'az', 'Basque' => 'eu', 'Bengali' => 'bn', 'Bosnian' => 'bs', 'Bulgarian' => 'bg', 'Catalan' => 'ca', 'Chinese Simplified' => 'zh-CN', 'Chinese Traditional' => 'zh-TW', 'Croatian' => 'hr', 'Czech' => 'cs', 'Danish' => 'da', 'Dari' => 'fa-AF', 'Dutch' => 'nl', 'English (UK)' => 'en-GB', 'Estonian' => 'et', 'Filipino' => 'fil', 'Finnish' => 'fi', 'French' => 'fr', 'Galician' => 'gl', 'Georgian' => 'ka', 'German' => 'de', 'Greek' => 'el', 'Hebrew' => 'he', 'Hindi' => 'hi', 'Hungarian' => 'hu', 'Icelandic' => 'is', 'Indonesian' => 'id', 'Irish (Gaelic)' => 'ga-IE', 'Italian' => 'it', 'Japanese' => 'ja', 'Kazakh' => 'kk', 'Khmer' => 'km', 'Korean' => 'ko', 'Kurdish' => 'ku', 'Latvian' => 'lv', 'Lithuanian' => 'lt', 'Maori' => 'mi', 'Macedonian (FYROM)' => 'mk', 'Malay' => 'ms', 'Maltese' => 'mt', 'Marathi' => 'mr', 'Nepali' => 'ne-NP', 'Norwegian' => 'no', 'Norwegian Nynorsk' => 'nn-NO', 'Pashto' => 'ps', 'Persian' => 'fa', 'Polish' => 'pl', 'Portuguese' => 'pt-PT', 'Portuguese, Brazilian' => 'pt-BR', 'Romanian' => 'ro', 'Russian' => 'ru', 'Serbian (Cyrillic)' => 'sr', 'Sinhalese' => 'si-LK', 'Slovak' => 'sk', 'Slovenian' => 'sl', 'Spanish' => 'es-ES', 'Spanish, Mexico' => 'es-MX', 'Swedish' => 'sv-SE', 'Tamil' => 'ta', 'Thai' => 'th', 'Turkish' => 'tr', 'Ukrainian' => 'uk', 'Urdu (Pakistan)' => 'ur-PK', 'Uzbek' => 'uz', 'Vietnamese' => 'vi', 'Zulu' => 'zu'}
-
 #language{:style=>'width: 780px; padding: 0; margin: 0;'}
-  - languages.each_slice(6) do |group|
+  - fetch_crowdin_languages.each_slice(6) do |group|
     .language_row{:style=>'width: 780px; padding: 0; margin: 0;'}
-      - group.each do |name, code|
+      - group.each do |properties|
```

Comment:
If you are only interacting with one element of properties, then might as well enumerate through that list

Something like

group.collect {|g| g[:crowdin_code_s]}.each

Though I think there's a better way to write that